lib/ansible/cli/__init__.py

```diff
@@ -89,18 +89,24 @@ def initialize_locale():
 
 _internal.setup()
 
+from ansible.errors import AnsibleError, ExitCode
+
 try:
     from ansible import constants as C
     from ansible.utils.display import Display
     display = Display()
 except Exception as ex:
-    print(f'ERROR: {ex}\n\n{"".join(traceback.format_exception(ex))}', file=sys.stderr)
+    if isinstance(ex, AnsibleError):
+        ex_msg = ' '.join((ex.message, ex._help_text)).strip()
+    else:
+        ex_msg = str(ex)
+
+    print(f'ERROR: {ex_msg}\n\n{"".join(traceback.format_exception(ex))}', file=sys.stderr)
```

Comment:
he, was deep in rabbit hole trying to figure out why 'no context' still resolved in a 'true'  ... i forgot we have direct error print on config ...